base-template/base-template.md

```diff
@@ -2,20 +2,120 @@
 <!-- Describe the title of your article by replacing "Base Template" with the page name you want to publish to. -->
 # Base template
 
-## Prerequisites
-
 ## Overview
+The base template is a blueprint that outlines any type of template required for a specific documentation type. That is, the base template will guide you through the process of creating a well structured template that can be used to create/standardize multiple documents similar in content. 
+
+Commonly used documentation types that we come across in our routine work can be broadly categorized into how-to guides, tutorials, reference or explanation documents. Any document that you create is likely to either be one of these types or a combination of a few of them. With the help of the base template, you can create a uniform, regulated template for any of these documentation types.
+
+
+You are likely to require the base template to create a new template or modify an existing one,  if you have identified repetitive, similarly structured content in multiple documents that need to be uniformly structured and aligned to a standardized document. 
+
+
+## Before you start
```

Comment:
Under each heading, I'd be expecting to see writing guidance to the template author. Something like:
{All templates should include this section. It should list any skillsets or prior reading which would be expected from a document author who is about to write their own document against your template.}
or
{A template may optionally include this section, based on <some condition>.}

Then proceed to provide example text.